src/lib/proposer/proposer.ml

```diff
@@ -211,7 +211,7 @@ let run ~logger ~prover ~verifier ~trust_system ~get_completed_work
       let log_bootstrap_mode () =
         Logger.info logger ~module_:__MODULE__ ~location:__LOC__
           "Bootstrapping right now. Cannot generate new blockchains or \
```

Comment:
`Bootstrap process currently in progress -- unable to produce new blocks until bootstrapping is completed`

Reply:
I prefer the message I wrote (which is now in the LHS) to the one currently in the PR, but @o1pranay's message is even better than that one because it mentioned the reason no frontier is available is bootstrap.